### DISCUSSION TITLE
End of life phase of the project

### DISCUSSION THREAD

**Discussion by Ein-Tim** [corona-warn-app/cwa-website#3304] [Q&A] [answered]
As [announced](https://github.com/corona-warn-app/cwa-website/pull/3302#issuecomment-1369019402) by @GisoSchroederSAP, the ramp down period of the project has begun. Some questions:
1. What changes can we expect on the mobile app in this period (bug fixes only, feature changes, etc.)?
	1. If there won't be any new features implemented, will you close the feature request issues in the https://github.com/corona-warn-app/cwa-wishlist repository?
2. Will something change here on GitHub?

Now more general questions:
1. How is the shutdown of the infrastructure planned (everything at one date or will more and more parts of the infrastructure will be shutdown in waves)?
2. Is there an official date where the project will be sunsetted?

**Comment by MikeMcC399** [comment, marked as answer]
- I added an issue https://github.com/corona-warn-app/cwa-wishlist/issues/876, however that has not received any feedback.
- According to https://github.com/corona-warn-app/cwa-app-android/pull/5885 the app will not be developed further after June 1, 2023.
- I also got feedback in https://github.com/corona-warn-app/cwa-map-public-frontend/issues/35#issuecomment-1457669517 that https://map.schnelltestportal.de/ is preparing shutdown.

**Comment by GisoSchroederSAP** [comment]
Good evening, 
We totally understand your passion and your information request.
Due to the highly sensitive political subject and due to contractual dependencies in the steps towards the end of CWA's development activities, we cannot yet answer individual questions separately here. Instead, we are now preparing a new FAQ section around this theme, which hopefully will answer all your questions (and even more questions). Please stay curious and patient - thank you for that!

**Comment by Ein-Tim** [comment]
FYI: Regarding the end of tests, etc.: have a look at https://twitter.com/laborostsachsen/status/1633752242682511360?s=20

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

Hopefully you will not take too long in publishing some information about changes.

The change to the legal framework [TestV](https://www.buzer.de/gesetz/14991/v292062-2023-03-01.htm) on March 1, 2023 already has a major impact on the app regarding testing and so far there has been no blog update or change to the FAQs to explain the change.

**Reply by GisoSchroederSAP** [reply]
That is so true, I totally understand, @MikeMcC399 . 
However, please note that this change to the legal framework TestV is outside our responsibility; and it was not at all communicated to us, not even indicated. 
So, we cannot fix miscommunication somewhere, but we stick to our plans here: To stay consistent with the app implementation - regarding _wording_ and as well regarding _dates_.

**Reply by MikeMcC399** [reply]
The planned change to TestV was already published on November 24, 2022 to take effect on March 1, 2023. See [Fünfte Verordnung zur Änderung der Coronavirus-Testverordnung](https://www.bundesanzeiger.de/pub/publication/wXbxqfkUdLM85aXaEg9/content/wXbxqfkUdLM85aXaEg9/BAnz%20AT%2024.11.2022%20V2.pdf?inline).

The RKI page https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/WarnApp/Warn_App.html hasn't been updated since July 2022.
The Twitter channel https://twitter.com/coronawarnapp last update was Feb 1, 2023.
The last Blog update on https://www.coronawarn.app/en/blog/ was for CWA 3.1 on Feb 27, 2023.

So I agree that there has been miscommunication leading to no communication on this aspect to users so far.

**Comment by GisoSchroederSAP** [comment]
Unfortunately, you're right again, @MikeMcC399.
Still, this communication does not reflect our planning with app and with the communication on the web, in the FAQ or in GitHub.
We try our best to stay consistent and transparent, being fully aware, that our community expects more.
